### PR TITLE
Fix ArrayIndexOutOfBoundsException in PersonTravelTimeComputation #3270

### LINKED ISSUE #3234: Error trying to create a graph ArrayIndexOutOfBoundsException: -1
it happens every now and then!

`18:09:31.235 ERROR b.a.p.GraphsStatsAgentSimEventsListener - Error trying to create a graph
java.lang.ArrayIndexOutOfBoundsException: -1
	at java.util.ArrayList.elementData(ArrayList.java:422)
	at java.util.ArrayList.get(ArrayList.java:435)
	at beam.analysis.plots.PersonTravelTimeAnalysis$PersonTravelTimeComputation.compute(PersonTravelTimeAnalysis.java:69)
	at beam.analysis.plots.PersonTravelTimeAnalysis$PersonTravelTimeComputation.compute(PersonTravelTimeAnalysis.java:54)
	at beam.analysis.plots.PersonTravelTimeAnalysis.compute(PersonTravelTimeAnalysis.java:165)
	at beam.analysis.plots.PersonTravelTimeAnalysis.createGraph(PersonTravelTimeAnalysis.java:142)
	at beam.analysis.plots.GraphsStatsAgentSimEventsListener.createGraphs(GraphsStatsAgentSimEventsListener.java:96)
	at beam.sim.BeamSim.$anonfun$notifyIterationEnds$4(BeamSim.scala:387)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)`

### PR DESCRIPTION
**Pull request by dgm90**
Fix https://github.com/LBNL-UCB-STI/beam/issues/3234
also revert this change https://github.com/LBNL-UCB-STI/beam/commit/a531d05a7b41034f35625c10aee00278c19ef32b#diff-fcd2a00c693c8e1382cda931e2204f3535b628669b88ad3338afd919c935bf91R12

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3271)

### CODE REVIEW

File: test/input/sf-light/sf-light-1k.conf [hunk 1]
```diff
@@ -9,7 +9,7 @@ include "../common/matsim.conf"
 # Agentsim
 ##################################################################
 beam.agentsim.simulationName = "sf-light-1k-xml"
-beam.agentsim.agentSampleSizeAsFractionOfPopulation = 0.001
+beam.agentsim.agentSampleSizeAsFractionOfPopulation = 1.0
```
Comment (dgm90) [inline test/input/sf-light/sf-light-1k.conf:12]: Revert the change from here https://github.com/LBNL-UCB-STI/beam/pull/3215/files#diff-fcd2a00c693c8e1382cda931e2204f3535b628669b88ad3338afd919c935bf91R12

**Comment by nikolayilyin** [comment]
test!

**Comment by dgm90** [comment]
Test!

**Review by nikolayilyin — APPROVED** [review]
lgtm
